planemo/tool_builder.py

```diff
@@ -19,7 +19,7 @@
 DEFAULT_CWL_VERSION = "v1.0"
 
 
-TOOL_TEMPLATE = """<tool id="{{id}}" name="{{name}}" version="{{version}}" python_template_version="3.5">
+TOOL_TEMPLATE = """<tool id="{{id}}" name="{{name}}" version="{{version}}+galaxy0" python_template_version="3.5" profile="21.05">
```

Comment:
We can also drop the `python_template_version`, that is implied by a recent profile